recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set version = "5.0.2" %}
 {% set major = version.rpartition('.')[0] %}
 {% set cuda_major = (cuda_compiler_version|default("11.8")).rpartition('.')[0] %}
-{% set build = 0 %}
+{% set build = 1 %}
```

Comment:
Can see there are some things happening with the `build/number` in this recipe

So not sure if just bumping here is sufficient. Or if something more needs to be done

Reply:
Ah got it. I hope this is all that is needed, otherwise the bot PRs have been wrong!